qubit-sdk/src/main/java/com/qubit/android/sdk/internal/placement/repository/PlacementAttributesRepository.kt

```diff
@@ -0,0 +1,10 @@
+package com.qubit.android.sdk.internal.placement.repository
+
+import com.google.gson.JsonObject
+
+interface PlacementAttributesRepository {
+
+  fun save(key: String, value: String)
```

Comment:
It is little bit weird, that saving works on String, while `load` loads JsonObject.

Reply:
Updated this class to operate only on Strings and introduced `PlacementAttributesInteractor` which is responsible for converting them from/to desired types.

Reply:
Hmm. It operates now on String and *, which is even worse than before.
I tought about using JsonObject in both directions.

When you call storing attributes in EventTrackerImpl instead of using EventModel, which mirrors database table (that's why there is body as string), you can use `qbEvent`.

EventTrackerImpl:
```
   placementAttributesInteractor.storeEventAttribute(qbEvent);
```

PlacementAttriubutesInteractor:
```
  override fun storeEventAttribute(event: QBEvent) {
    mapTypeToKey(event.type)?.let {
      placementAttributesRepository.save(it, event.toJsonObject())
    }
  }
```

PlacementAttributesRepository:
```
  fun save(key: String, value: JsonObject)
  fun load(): MutableMap<String, JsonObject>
```

Reply:
fixed